mappings/net/minecraft/util/math/Vec3i.mapping

```diff
@@ -96,3 +96,9 @@ CLASS net/minecraft/class_2382 net/minecraft/util/math/Vec3i
 	METHOD method_35861 north ()Lnet/minecraft/class_2382;
 	METHOD method_35862 multiply (I)Lnet/minecraft/class_2382;
 		ARG 1 scale
+	METHOD method_39675 createRangeValidator (I)Ljava/util/function/Function;
+		ARG 0 maxAbsValue
+	METHOD method_39676 (ILnet/minecraft/class_2382;)Lcom/mojang/serialization/DataResult;
+		ARG 1 vec
+	METHOD method_39677 createCodec (I)Lcom/mojang/serialization/Codec;
```

Comment:
How is this different from `Vec3i#CODEC`? Maybe `createOffsetCodec` since this method is used for offset field in block predicates.

```suggestion
	METHOD method_39677 createOffsetCodec (I)Lcom/mojang/serialization/Codec;
```